### DISCUSSION TITLE
Left navigation bar

### DISCUSSION THREAD

**Discussion by zhangwh807** [mkdocs/mkdocs#3611] [Q&A] [answered]
I set up the navigation directory structure with mkdocs.yml. When I build readthedocs, clicking on any directory in the left navigation bar will open all the directories in the left navigation bar at the same time

**Comment by zhangwh807** [comment]
How do you solve this? I just want to open the directory that I clicked on

**Reply by pawamoy** [reply]
Please provide a minimum reproducible example :slightly_smiling_face: 
It's difficult to help when we don't know what you are doing exactly.

**Comment by zhangwh807** [comment]
I feel that the theme style also has a lot to do with it.I gave it a try and if that's the case there's no problem
“theme:
  name: cerulean
  nav_style: dark”
But if I write "theme: readthedocs," I get a problem with the left navigation

**Comment by oprypin** [comment, marked as answer]
* This will be fixed by https://github.com/mkdocs/mkdocs/pull/3464
